src/realm/parser/generated/query_flex.cpp

```diff
@@ -101,6 +101,7 @@ typedef int16_t flex_int16_t;
 typedef uint16_t flex_uint16_t;
 typedef int32_t flex_int32_t;
 typedef uint32_t flex_uint32_t;
+typedef uint64_t flex_uint64_t;
```

Comment:
please do not update these generated files unless you are changing the query parser